bind_uploader.go

```diff
@@ -74,7 +71,7 @@ func (bu *bindUploader) uploadStreamInternal(inputStream *bytes.Buffer, compress
 	}
 
 	// use a placeholder for source file
-	putCommand := fmt.Sprintf("put 'file:///tmp/placeholder' '%v' overwrite=true", stageName)
+	putCommand := fmt.Sprintf("put 'file:///tmp/placeholder/%v' '%v' overwrite=true", dstFileName, stageName)
```

Comment:
what is the safety guard of '/tmp/placeholder'? creating, cleaning, multi put, etc.

Reply:
'placeholder' is just meant to retrieve the appropriate credentials and stage data to upload to. The stage created will persist throughout the session until it ends. Also it's important to note that this PUT is internal for binding arrays and different stages created throughout the session by a customer via PUT should not interact with this stage.